android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/AncLoginService.kt

```diff
@@ -19,16 +19,49 @@ package org.smartregister.fhircore.anc.ui.login
 import android.content.Intent
 import javax.inject.Inject
 import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpLoginActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpSetupActivity
 import org.smartregister.fhircore.engine.ui.login.LoginActivity
 import org.smartregister.fhircore.engine.ui.login.LoginService
+import org.smartregister.fhircore.engine.util.FORCE_LOGIN_VIA_USERNAME
+import org.smartregister.fhircore.engine.util.SharedPreferencesHelper
 
-class AncLoginService @Inject constructor() : LoginService {
+class AncLoginService @Inject constructor(val sharedPreferencesHelper: SharedPreferencesHelper) :
+  LoginService {
 
   override lateinit var loginActivity: LoginActivity
 
-  override fun navigateToHome() {
+  override fun navigateToHome(canSetOtp: Boolean) {
+    // Todo: check whether to setup PIN     or moveTo Register Home
```

Comment:
Change from //TODO to documentation. //TODO makes it seem like something is yet to be done.

Reply:
updated